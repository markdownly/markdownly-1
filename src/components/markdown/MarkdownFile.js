import React from 'react';
import PropTypes from 'prop-types';

export default function MarkdownFile({ markdownTitles }) {
  console.log('thiss', markdownTitles);
  const titleList = markdownTitles.map((title) => {
    return <li key={title}>{title}</li>;
  });
  return (
    <ul>{titleList}</ul>
  );

}

MarkdownFile.propTypes = {
  markdownTitles: PropTypes.array.isRequired
};
