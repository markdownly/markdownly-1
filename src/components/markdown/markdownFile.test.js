import React from 'react';
import renderer from 'react-test-renderer';
import MarkdownFile from './MarkdownFile';

describe('MarkdownFile', () => {
  it('matches a snapshot', () => {
    const markdownTitles = ['file1', 'file2'];

    const tree = renderer.create(
      <MarkdownFile markdownTitles={markdownTitles} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
