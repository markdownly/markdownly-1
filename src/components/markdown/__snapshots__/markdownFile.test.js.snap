// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkdownFile matches a snapshot 1`] = `
<ul>
  <li>
    file1
  </li>
  <li>
    file2
  </li>
</ul>
`;
